packages/wonder-blocks-core/src/hooks/__tests__/use-render-state.test.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import * as React from "react";
+import {renderHook as renderHookOnServer} from "@testing-library/react-hooks/server";
+import {renderHook} from "@testing-library/react-hooks";
```

Comment:
I like this a little better than what we did in some of the other hook tests.  It makes it clearer which tests are server-side tests vs. client-side tests.  It also makes the tests more consistent b/c they're rendering the hook using the same API.